packages/@react-aria/interactions/docs/useFocusVisible.mdx

```diff
@@ -74,3 +74,17 @@ function Example() {
   );
 }
 ```
+
+## Advanced
+
+<PageDescription>{docs.exports.addWindowFocusTracking.description}</PageDescription>
+<FunctionAPI function={docs.exports.addWindowFocusTracking} links={docs.links} />
+<br />
+Sometimes apps render portions of their tree into an iframe. In this case, we cannot accurately track if the focus
+is visible because we cannot see interactions inside the iframe. If you have this in your application's architecture
```

Comment:
Super-nit:

```suggestion
is visible because we cannot see interactions inside the iframe. If you have this in your application's architecture,
```